app/src/androidTest/java/com/github/se/eventradar/qrCode/qrCodeScanTicketUiTest.kt

```diff
@@ -91,6 +90,117 @@ class QrCodeScanTicketUiTest : TestCase(kaspressoBuilder = Kaspresso.Builder.wit
     mDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
   }
 
+  @Test
+  fun screenDisplaysNavigationElementsCorrectly() = run {
+    every { mockViewModel.uiState } returns myEventUiState
+    onComposeScreen<QrCodeScanTicketUiScreen>(composeTestRule) {
+      composeTestRule.setContent { QrCodeTicketUi(mockViewModel, mockNavActions) }
+      bottomNavMenu.assertIsDisplayed()
+      goBackButton.assertIsDisplayed()
+    }
+  }
+
+  @Test
+  fun screenDisplaysContentElementsCorrectly1() = run {
+    every { mockViewModel.uiState } returns myEventUiState
+    onComposeScreen<QrCodeScanTicketUiScreen>(composeTestRule) {
+      composeTestRule.setContent { QrCodeTicketUi(mockViewModel, mockNavActions) }
+      lazyEventDetails.assertIsDisplayed()
+      eventTitle {
+        assertIsDisplayed()
+        assertTextContains("Test_Event")
+      }
+      eventImage { assertIsDisplayed() }
+      descriptionTitle { assertIsDisplayed() }
+      descriptionContent {
+        assertIsDisplayed()
+        assertTextContains("Test Description")
```

Comment:
What do you think of putting directly `myEventUiState.value.eventUiState.eventName` instead of copying manually what the value needs to be ? I think it would be easier to maintain